sklearn/ensemble/_voting.py

```diff
@@ -438,7 +439,7 @@ def transform(self, X):
         Returns
         -------
         predictions
-            array-like of shape (n_samples, n_classifiers), being
+            Array-like of shape (n_samples, n_classifiers), being
```

Comment:
no actually this should be,
```
predictions : array-like of shape (n_samples, n_classifiers)
    Values predicted by each regressor.
```
(with the right indentation)

Reply:
out of curiosity, why should this be an `array-like` instead of an `array`. Can it output anything other than `ndarray`s? (My only guess would be that if a dataframe was input it could output one and that dfs fall under the array-like definition, but haven't followed in detail the inclusion of dfs as possible inputs)

Reply:
You are right, it can only return an `array`. We use `array-like` for input dtypes, and it was probably copy-pasted there...